sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationClientTest.java

```diff
@@ -2,23 +2,26 @@
 // Licensed under the MIT License.
 package com.azure.data.appconfiguration;
 
+import com.azure.core.http.rest.PagedIterable;
```

Comment:
Was this auto-sorted by IntelliJ? Curious because it's not lexical.

Reply:
not sure how intellij did it, I will manually fix it for now